content/md/supplier-onboarder/use-the-product-grid.md

```diff
@@ -0,0 +1,210 @@
+  2. The working locale selected on the product grid page stays the same when you open or edit a product form.
+  3. There are two types of locale, the working locale for your products and the user interface locale, which is the language you chose to work on your Onboarder.
+:::
+-->
+
+# Product grid view
+
+A `Gallery` view of the product grid is available. This view can be used to quickly identify and update products thanks to their image.
+
+With the `Gallery` view, you can see the following properties in the grid:
+- Supplier  reference
+- Label: attribute used as "label" in the family
+- Image: attribute used as "main picture" in the family
+- Completeness
+<!--
+- Number of complete variant products out of the number of variant products for a product model (ex: "2/3" means that 2 variants out of 3 for the product model are complete)
+-->
+
+:::info
+Other grid features also apply to this view but are not displayed (channel, locale and categories selection, filters, pagination, product selection, etc.)
+:::
+
+To use the `Gallery` view:
+1.  Click on `List`
+  ![Select the product grid view](../img/Products_display.png)
+1.  Select the view `Gallery`
+  ![Gallery view of the product grid](../img/Products_gallery_display.png)
+
+:::info
+When you change your working locale, you keep all your selected filters active.
+:::
+
+# Product grid pagination
+
+The product grid displays 25 products per page. To browse the product's pages, click on the number of the page you want to reach.
+
+:::info
+The grid can display up to 400 pages (10000 products). However, you can use the filters in the grid to optimize the results displayed.
+:::
+
+![Select the page you want to reach by clicking on the number of the page](../img/Products_views_click_on_pages.png)
+
+# Category panel
+
+## Browse products' categories
+
+On the left side of the page, you will see the category panel. Thanks to this panel, you can easily navigate between category trees or between categories.
+
+As a supplier, you can define your own category tree to easily find your products. The categories you define on your side won't be sent to the retailer.
+
+![Browse products' categories](../img/Products_Category1.png)
+
+## View unclassified products
+
+Below the category tree, you will see an additional category: `Unclassified products`. Click on this category to see all unclassified products. This category is added automatically and cannot be removed or edited.
+
+![View unclassified products](../img/Products_Category3.png)
+
+
+## View all products
+
+Click on `All products` to see all products (categorized and uncategorized) of the selected category tree.
+
+![View all products](../img/Products_Category4.png)
+
+## Product counter
+
+Onboarder displays the number of products available in each category and category tree. You will see a number into brackets nearby each category; this figure represents the total number of products classified in this category or sub-category.
```

Comment:
Do we talk about Onboarder or The Onboarder? If it's the latter, The is missing here!